frontend/src/app/modules/auth/login-page/login-page.component.scss

```diff
@@ -1,27 +1,33 @@
 .to-login-container {
-    margin-top: 10%;
+    margin-top: 6%;
     h2 {
         color: #732939;
         font-size: 26px;
         font-weight: 600;
         margin-bottom: 0;
         font-family: "Poppins", sans-serif;
     }
-    .already-registered {
+    .dont-have-account {
         margin: 0;
-        margin-top: 3%;
+        margin-top: 4%;
         color: #777;
         font-size: 13px;
         font-weight: 500;
-        span {
-            color: #732939;
-            font-weight: 600;
+        button{
```

Comment:
Maybe some animation at least besides changing the cursor style?
Opacity, highlighting, underlining...

Reply:
It's not really needed, loom also doesn't have it

Reply:
It doesn't mean that what they have looks nice

Reply:
Added color change when hovering